The target LDFLAGS have been ignored. Tools like torque
have been loaded from system libraries, even if a native
one was the target.
|$ ldd torque 
|    libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1
|    libcrypto.so.1.1 => /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
|    libssl.so.1.1 => /usr/lib/x86_64-linux-gnu/libssl.so.1.1
|    libicui18n.so.63 => not found
|    libicuuc.so.63 => not found
...

Signed-off-by: Andrej Valek <andrej.valek@siemens.com>

diff -Naur node-v10.15.1/deps/v8/gypfiles/toolchain.gypi node-v10.15.1/deps/v8/gypfiles/toolchain.gypi
--- node-v10.15.1/deps/v8/gypfiles/toolchain.gypi	2019-03-18 15:01:39.000000000 +0100
+++ node-v10.15.1/deps/v8/gypfiles/toolchain.gypi	2019-03-18 15:04:08.628361308 +0100
@@ -1106,6 +1106,7 @@
             'cflags': [ '-fno-strict-aliasing' ],
           }],
         ],  # conditions
+        'ldflags+': [ '$(LDFLAGS)' ],
       }],
       ['OS=="solaris"', {
         'defines': [ '__C99FEATURES__=1' ],  # isinf() etc.
